vsts/vsts.yaml

```diff
@@ -538,7 +538,7 @@ jobs:
           instanceUrlForTsaV2: 'MSAZURE'
           projectNameMSAZURE: 'One'
           areaPath: 'One\IoT\Platform and Devices\IoT Devices\SDKs\Managed'
-          iterationPath: 'One\IoT\Backlog'
+          iterationPath: 'One\Custom\IoT\Backlog'
```

Comment:
Interesting, I thought they wanted it to be set to the root of the project iteration path aka `One`

Reply:
That wouldn't be under IoT path, though. I just moved it to the renamed path.